teams/all.toml

```diff
@@ -42,3 +42,15 @@ excluded-people = [
     "celaus",
     "opeolluwa",
 ]
+
+# Private channel with all team members, so that we can have an easy way of reaching them.
+[[zulip-streams]]
+name = "rust project"
```

Comment:
I would make this `all/private` (which matches the naming of `all@rust-lang.org`. Then you could have an `all` (if it seems helpful) in the future.